### DISCUSSION TITLE
Markdown links: Is there a plugin that uses first #heading as link text

### DISCUSSION THREAD

**Discussion by zach-at-healthpilot** [BryanJenksCommunity/FAQ#103] [Q&A] [answered]
For markdown links, Is there a plugin that uses first #heading as link text? Otherwise i'm having to change a bunch of `[index](../blob/index.md)` autocompletes, where index.md looks like

# Cool title

Cool text

**Comment by tallguyjenks** [comment, marked as answer]
its native functionality, for markdown links just add a `#` right before the `)` in your link and it should allow you to link to headings. i personally prefer the wiki syntax and with those you can just type `[[#` and it will allow you to link to headings in that document and if you type `[[##` it will allow you to link to headings in any document and start a text search
